injectproxy/routes.go

```diff
@@ -292,24 +294,39 @@ func (r *routes) enforceLabel(h http.HandlerFunc) http.Handler {
 	})
 }
 
-// getLabelValue returns the statically set label value, or the label value
-// sent through a URL parameter.
-// It returns an error when either the value is found in both places, or is not found at all.
-func (r *routes) getLabelValue(req *http.Request) (string, error) {
-	formValue := req.FormValue(r.label)
-	if r.labelValue != "" && formValue != "" {
-		return "", fmt.Errorf("a static value for the %s label has already been specified", r.label)
+// getLabelValues returns a slice of strings of the statically set label value,
+// or of the label value sent through a URL parameter.
+// It returns an error when the URL parameter cannot be parsed,
+// or the value is found in both places, or is not found at all,
+// or one of the provided parameters is empty.
+func (r *routes) getLabelValues(req *http.Request) ([]string, error) {
+	err := req.ParseForm()
+	if err != nil {
+		return []string{}, fmt.Errorf("the query parameter can not be parsed: %w", err)
```

Comment:
(nit) and the same for all error returns

```suggestion
		return nil, fmt.Errorf("the query parameter can not be parsed: %w", err)
```